webpack.config.js

```diff
@@ -11,16 +11,23 @@ module.exports = {
     publicPath: "dist",
   },
   devServer: {
+    before: app => {
+      app.get("/view", (req, res) => {
+        res.sendFile("index.html", { root: "." });
+      });
+
+      app.get("/test", (req, res) => {
```

Comment:
out of curiosity, where does this fetch come from?

Reply:
When running tests, the thread front dynamically creates a script element whose source will be fetched via this url.  See initializeToolbox in thread.js